src/server/Generators/classes/pgSqlProvider.js

```diff
@@ -70,6 +70,23 @@ class PgSqlProvider {
 
     return query;
   }
+
+  parameterize() {
```

Comment:
I see you include a pg refactor in your mssql PR =P

Reply:
@laueian  I had to do it to support the async connection initialization for the mssql library